iron-list.html

```diff
@@ -589,7 +637,12 @@
     updateViewportBoundaries: function() {
       var scrollerStyle = window.getComputedStyle(this._scroller);
       this._scrollerPaddingTop = parseInt(scrollerStyle['padding-top'], 10);
-      this._viewportSize = this._scroller.offsetHeight;
+      this._viewportHeight = this._scroller.offsetHeight;
+
+      // If in grid mode compute the _viewportWidth and _physicalItemsPerRow
+      if (this.grid) {
+        this._updateGridMetrics();

```

Comment:
`_updateGridMetrics` belongs to `_updateMetrics`. I think `_updateGridMetrics` should only be called or maybe merged into `_updateMetrics` wdyt?